src/assets/wise5/components/match/bucket.ts

```diff
@@ -0,0 +1,12 @@
+import { Choice } from './choice';
+
+export class Bucket {
+  id: string;
+  items?: Choice[];
```

Comment:
Are there buckets that will never have any items?
```suggestion
  items: Choice[];
```

Reply:
The bucket object in the authored content does not have items.